packages/@react-aria/overlays/src/usePreventScroll.ts

```diff
@@ -11,7 +11,7 @@
  */
 
 import {chain, getScrollParent} from '@react-aria/utils';
-import {useLayoutEffect} from 'react';
+import {useEffect, useLayoutEffect} from 'react';
```

Comment:
Could you try with `import {chain, getScrollParent, useLayoutEffect} from '@react-aria/utils';`? It should do a roughly similar thing https://github.com/adobe/react-spectrum/blob/main/packages/%40react-aria/utils/src/useLayoutEffect.ts
Maybe it needs updating to this? I don't know quite enough about SSR personally.

Reply:
That works as well. Done 👍